dts/arm/nanoxplore/ng/ngultra.dtsi

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2022 NanoXplore
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <arm/armv8-r.dtsi>
+#include <dt-bindings/interrupt-controller/arm-gic.h>
+#include <mem.h>
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "arm,cortex-r52";
+			reg = <0>;
+		};
+	};
+
+	timer {
+		compatible = "arm,armv8-timer";
+		interrupt-parent = <&gic>;
+		interrupts = <GIC_PPI 13 IRQ_TYPE_LEVEL
+			      IRQ_DEFAULT_PRIORITY>,
+			     <GIC_PPI 14 IRQ_TYPE_LEVEL
+			      IRQ_DEFAULT_PRIORITY>,
+			     <GIC_PPI 11 IRQ_TYPE_LEVEL
+			      IRQ_DEFAULT_PRIORITY>,
+			     <GIC_PPI 10 IRQ_TYPE_LEVEL
+			      IRQ_DEFAULT_PRIORITY>;
```

Comment:
nit: these entries should fit in 80 columns, can you unindent them? makes it easier to read

Reply:
Indeed this code snippet fits in the 80 columns even without the line jump.
Is it ok for you?

Reply:
yeah I think readability would improve a fair bit